content/@i18n/ja/concepts/tokens/freezes.md

```diff
@@ -1,7 +1,7 @@
 ---
 html: freezes.html
 parent: tokens.html
-blurb: 発行者はコンプライアンス目的でトークンの取引を停止できます。
+blurb: 凍結では、コンプライアンス目的で発行済み通貨の取引を停止できます。
```

Comment:
This change (unintentionally?) reverts the changes done in https://github.com/XRPLF/xrpl-dev-portal/commit/27058687b36cdd2a45bb9af6a052252adcab23f9 . This PR should not change it.

Reply:
Reverted.